examples/teams/teams.json

```diff
@@ -37,17 +43,18 @@
         "RoleNames": [
           "GROUP_CLUSTER_MANAGER"
         ],
-        "ApiKeys": {
-          "PublicKey": "{{resolve:secretsmanager:mongodb/atlas/secrets:SecretString:AtlasMongoDBPublicKey}}",
-          "PrivateKey": "{{resolve:secretsmanager:mongodb/atlas/secrets:SecretString:AtlasMongoDBPrivateKey}}"
+        "Profile": {
+          "Ref": "Profile"
         }
       }
     }
   },
   "Outputs": {
-    "Teams" : {
+    "Teams": {
       "Description": " teams",
-      "Value" : { "Ref" : "Teams" }
+      "Value": {
+        "Ref": "Teams"
+      }
```

Comment:
if you have tested,  will { "Ref":"Teams"} return all the properties under PrimaryIdentifier?

Reply:
testing now

Reply:
I only have "/properties/TeamId" in the private identifier so it returned only the teamId

![Screenshot 2023-02-22 at 09 49 33](https://user-images.githubusercontent.com/5663078/220584150-87dd4a92-4e37-426b-8050-5077d29727b7.png)

Reply:
yes, I see the TeamId and the profile name

![Screenshot 2023-02-22 at 10 03 39](https://user-images.githubusercontent.com/5663078/220587638-0fd2bbaf-d269-4a82-8249-ef375bf69f6b.png)